ucm2/USB-Audio/Rane/SL-1-HiFi.conf

```diff
@@ -0,0 +1,69 @@
+SectionDevice."Line1" {
+	Comment "Line out 1"
+
+	Value {
+		PlaybackPriority 100
+		PlaybackPCM "hw:${CardId},0"
+
+		PlaybackChannels 2
+		PlaybackMixerElem "Line out 1"
```

Comment:
This USB card does not have any controls (exported from the driver), so `PlaybackMixerElem` and `PlaybackVolume` / `PlaybackSwitch` lines should be removed.